tests/drivers/build_all/sensors_i_z.conf

```diff
@@ -31,4 +31,5 @@ CONFIG_TH02=y
 CONFIG_TI_HDC=y
 CONFIG_TMP007=y
 CONFIG_TMP112=y
+CONFIG_TMP116=y
```

Comment:
You also need to add macros to `tests/drivers/build_all/dts_fixup.h` to fix the shippable failure

Reply:
Already done and waiting. Thanks!